src/components/Person/Person.jsx

```diff
@@ -1 +1,16 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age = 0, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age === 0 ? '' : <p className="Person__age">I am {age}</p>}
```

Comment:
```suggestion
      {age > 0 && <p className="Person__age">I am {age}</p>}
```
You can use `&&` operator instead as you return nothing in other case